local-cli/generator-android/templates/src/app/src/main/AndroidManifest.xml

```diff
@@ -19,6 +13,7 @@
       <activity
         android:name=".MainActivity"
         android:label="@string/app_name"
+		android:screenOrientation="portrait"
```

Comment:
Because in iOS project by default only the portrait orientation is enable. I think it should match the same configuration.